src/Webflow.js

```diff
@@ -53,16 +53,14 @@ export default class Webflow {
     token,
     version = '1.0.0',
   } = {}) {
-    if (!token) throw buildRequiredArgError('token');
-
     this.responseWrapper = new ResponseWrapper(this);
 
     this.endpoint = endpoint;
     this.token = token;
 
     this.headers = {
       Accept: 'application/json',
-      Authorization: `Bearer ${token}`,
+      ...(token && { Authorization: `Bearer ${token}` }),
```

Comment:
How will the token be set if not passed in the constructor?

Reply:
It won't actually, the request in the issue is to make it completely optional. You can find more about the use-case in [this comment](https://github.com/webflow/js-webflow-api/pull/57#issuecomment-1147674149).

Reply:
Seems to me there should be a setter because there's not a scenario where you can call the API without a token

Reply:
There is a scenario when `endpoint` is not set to `DEFAULT_ENDPOINT`. It could be enforced by throwing a `buildRequiredArgError` if `endpoint !== DEFAULT_ENDPOINT` and `token` is falsy.